src/internal_handlers/peer_management/tester.rs

```diff
@@ -26,8 +53,7 @@ impl Tester {
             let (_announcement_handle, announcement_handler) =
                 AnnouncementHandler::new(peer_db.clone());
             message_handlers.add_handler(0, MessageHandler::new(announcement_handler));
```

Comment:
You can remove the announcement handler now that receive it and manage it in the handshake